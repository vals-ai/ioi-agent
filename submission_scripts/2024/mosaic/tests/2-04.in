version https://git-lfs.github.com/spec/v1
oid sha256:0c0c5d18f55f291cd9f6d3159c176e8add73e4634c211a681871745ac8e9316d
size 3528
