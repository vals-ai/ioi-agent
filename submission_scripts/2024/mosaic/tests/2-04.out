version https://git-lfs.github.com/spec/v1
oid sha256:112938b4bdf0b1e0894f06731fc7505084a3d2477c2d1608fb9a139145cc8a10
size 1114
